keyboards/handwired/derb_planck/rev1/info.json

```
@@ -0,0 +1,86 @@
+{
+    "manufacturer": "Bernd Lehmann",
+    "keyboard_name": "derb_planck",
+    "maintainer": "der-b",
+    "bootloader": "qmk-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "ps2": true
+    },
+    "matrix_pins": {
+        "cols": ["D0", "D1", "D3", "D4", "D6", "D7", "B4"],
+        "rows": ["B3", "B1", "F0", "F1", "B5", "B6", "C6"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://der-b.com/posts/2024-05-08-mechanical-keyboard-derb-planck/",
+    "ps2": {
+        "mouse_enabled": true,
+        "driver": "usart"
+    },
+    "usb": {
+        "no_startup_check": true,
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
```

add Community Layout support
```suggestion
    "community_layouts": ["ortho_4x12"],
    "layouts": {
```